plugins/woocommerce-admin/client/wp-admin-scripts/tags-tracking/index.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import { recordEvent } from '@woocommerce/tracks';
+
+const addNewAttribute = document.querySelector( '#addtag #submit' );
+
+function actionButtonEventHandler( event ) {
+	const actionClass = event.target.parentElement.classList[ 0 ];
+
+	const actions = {
+		edit: 'edit',
+		inline: 'quick_edit',
+		delete: 'delete',
+		view: 'preview',
+	};
+
+	if ( ! actions[ actionClass ] ) {
+		return;
+	}
+
+	recordEvent( 'product_tags_list_action_click', {
+		selected_action: actions[ actionClass ],
+	} );
+}
+
+function addActionButtonListeners() {
+	const actionButtons = document.querySelectorAll( '.row-actions span' );
+	actionButtons.forEach( ( button ) => {
+		button.removeEventListener( 'click', actionButtonEventHandler );
+		button.addEventListener( 'click', actionButtonEventHandler );
+	} );
+}
+addActionButtonListeners();
+
+addNewAttribute?.addEventListener( 'click', function () {
+	const archiveInput = document.querySelector( '#attribute_public' );
+	const sortOrder = document.querySelector( '#attribute_orderby' );
+	recordEvent( 'product_tags_add', {
+		enable_archive: archiveInput?.checked ? 'yes' : 'no',
+		default_sort_order: sortOrder?.value,
```

Comment:
I think that these props aren't necessary here. I think we can also delete the two const declared above.

Reply:
@octaedro Pedro did request for these props in the original issue: https://github.com/woocommerce/woocommerce/issues/32676, is there a particular reason you don't think they are necessary?

Reply:
Hey @louwie17,
I think we shouldn't track those props because `Tags` don't have an order or archives.
Maybe there was a misunderstanding. Looking at the original issue, I couldn't find those props either

![screenshot-github com-2022 05 27-16_04_31](https://user-images.githubusercontent.com/1314156/170775044-56aa647a-0e97-4dcd-b256-22253ec13378.png)

Reply:
Aaah sorry, you are right, I missed that this was for the `tags` track. I automatically thought it was the category add track. Good catch!!

Reply:
I removed it as part of : https://github.com/woocommerce/woocommerce/pull/33118/commits/96996a202a12ef2c6c96ccd3428a04853a3034d7